packages/data/addon/serializers/facts/interface.ts

```diff
@@ -10,7 +10,7 @@ import NaviFactResponse from 'navi-data/models/navi-fact-response';
 type ResponsePayload = any;
 
 export interface ResponseV1 {
-  readonly rows: Array<Record<string, unknown>>;
+  readonly rows: Array<Record<string, string | number | boolean>>;
```

Comment:
This could maybe be `string | number`, but I'm in favor of keeping it `unknown` since this is our generic response type so we really don't know what an api will give us
```suggestion
  readonly rows: Array<Record<string, >>;
```